manage.py

```diff
@@ -117,31 +122,43 @@ def init_db(skip_create_db=False):
 def init_postgres(db_uri):
     """Initializes PostgreSQL database from provided URI.
 
-    New user and database will be created, if needed. It also creates uuid-ossp extension.
+    New user and database will be created,
+    if needed. It also creates uuid-ossp extension.
     """
     hostname, port, db, username, password = data_utils.explode_db_uri(db_uri)
     if hostname not in ['localhost', '127.0.0.1']:
         raise Exception('Cannot configure a remote database')
 
     # Checking if user already exists
-    retv = subprocess.check_output('sudo -u postgres psql -t -A -c "SELECT COUNT(*) FROM pg_user WHERE usename = \'%s\';"' % username, shell=True)
+    retv = subprocess.check_output('sudo -u postgres psql -t -A -c '
+                                   '"SELECT COUNT(*) FROM pg_user WHERE '
+                                   'usename = \'%s\';"' % username, shell=True)
```

Comment:
Is `usename` here a typo? (Yes, I realise that the typo also exists in the source, so I'd say this should be fixed in its own commit if it is a mistake to begin with.)